templates/terraform/examples/region_health_check_http_logs.tf.erb

```diff
@@ -1,6 +1,4 @@
 resource "google_compute_region_health_check" "<%= ctx[:primary_resource_id] %>" {
-  provider = google-beta
```

Comment:
This example relies on the beta-only field log_config, which causes the example to fail